src/flows/shared/steps/OrganizeCollection/Editor/StagingArea.tsx

```diff
@@ -331,6 +333,16 @@ function StagingArea({ tokensRef }: Props) {
 
   const sensors = useSensors(useSensor(MouseSensor), useSensor(TouchSensor));
 
+  const activeSectionId = useActiveSectionIdState();
+  const handleBackspacePress = useCallback(() => {
+    if (activeSectionId) {
+      const itemIds = stagedCollectionState[activeSectionId]?.items.map(({ id }) => id);
+      deleteSection(activeSectionId, itemIds);
```

Comment:
what if `itemIds` is undefined? might be a bit safer to add:

`const itemIds = staged... ?? []`

Reply:
good call, done!